frame/contracts/src/benchmarking/mod.rs

```diff
@@ -3020,10 +3009,8 @@ benchmarks! {
 	print_schedule {
 		#[cfg(feature = "std")]
 		{
-			let weight_limit = T::DeletionWeightLimit::get();
-			let max_queue_depth = T::DeletionQueueDepth::get() as usize;
-			let empty_queue_throughput = ContractInfo::<T>::deletion_budget(0, weight_limit);
-			let full_queue_throughput = ContractInfo::<T>::deletion_budget(max_queue_depth, weight_limit);
+			let empty_queue_throughput = ContractInfo::<T>::deletion_budget(Weight::MAX);
+			let full_queue_throughput = ContractInfo::<T>::deletion_budget(Weight::MAX);
```

Comment:
1. There is no difference between empty and full queue anymore.
2. We should pass the block weight limit here instead of `Weight::MAX`. Should be defined on the `frame_system` config trait.

Reply:
🤦